examples/matrix/src/matrix.c

```diff
@@ -0,0 +1,124 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Is this original code? can we just vendor existing code for those kind of examples?